### DISCUSSION TITLE
ctx - Python library compromise - "likely to mine AWS credentials"

### DISCUSSION THREAD

**Discussion by jackiekazil** [Open-Source-Software-Neighborhood-Watch/town-square#13] [General]
https://twitter.com/s0md3v/status/1529005758540808192?s=27&amp;t=yotaLA-kPIbdW0qdGf715A 

https://isc.sans.edu/forums/diary/ctx+Python+Library+Updated+with+Extra+Features/28678/

https://pypi.org/project/ctx/ 

https://python-security.readthedocs.io/pypi-vuln/index-2022-05-24-ctx-domain-takeover.html

**Comment by tpike3** [comment]
Oh wow this is fascinating and gives several vectors for possible detections from age, to code specifics.  It seems the [code nutrition](https://github.com/Open-Source-Software-Neighborhood-Watch/OSSNutritionLabelPrototypes) labels could help as well

**Reply by gps-iqt-labs** [reply]
Absolutely @tpike3 -- there's actually a group at UMD ARLIS [exploring the viability of that approach](https://github.com/UMD-ARLIS/DevSecOps_OpenSource/wiki/Project-Plan#approach). RE: other possible detections, don't forget [AuraBorealis](https://github.com/IQTLabs/AuraBorealisApp).
